platform/platform.hpp

```diff
@@ -122,8 +123,12 @@ class Platform
   /// Client app should not replace default resource dir.
   void SetResourceDir(string const & path);
 
-  /// Creates directory at filesystem
-  EError MkDir(string const & dirName) const;
+  /// Creates directory in the filesystem.
+  static EError MkDir(string const & dirName);
```

Comment:
Do we need for uniformity add WARN_UNUSED_RESULT here too?

Reply:
Won't hurt.
done